packages/cubejs-databricks-jdbc-driver/src/helpers.ts

```diff
@@ -15,26 +15,37 @@ async function fileExistsOr(
 
 export async function resolveJDBCDriver(): Promise<string> {
   return fileExistsOr(
-    path.join(process.cwd(), 'DatabricksJDBC42.jar'),
+    path.join(process.cwd(), `databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`),
     async () => fileExistsOr(
-      path.join(__dirname, '..', 'download', 'DatabricksJDBC42.jar'),
+      path.join(__dirname, '..', 'download', `databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar`),
       async () => {
         const pathOrNull = await downloadJDBCDriver();
         if (pathOrNull) {
           return pathOrNull;
         }
         throw new Error(
-          'Please download and place DatabricksJDBC42.jar inside your ' +
+          `Please download and place databricks-jdbc-${OSS_DRIVER_VERSION}-oss.jar inside your ` +
```

Comment:
I'm not a software lawyer, but
Can we just put jar inside npm package? Download it during build, and distribute together.
IIUC, it is distributed under Apache license
It does contain some native code for lz4 in `com.databricks.internal.jpountz.util`, but, given that they distribute it as a single jar with all variants inside, I think it's gonna be fine regarding compatibility with target arch/os/...

Reply:
Yep, okay